stepup/README.md

```diff
@@ -30,36 +30,34 @@ surfnet_yubikey_api_client:
     client_secret: 'YOUR_SECRET'
 ```
 
-You should then get the apps initialised
 You can then bring up the containers using docker compose:
 
-Initialise the middelware database:
-```
-docker compose exec middleware /var/www/html/bin/console  doctrine:migrations:migrate --env=prod --em=deploy
+```text
 docker-compose up -d
 ```
 
-Then the webauthn db
-```
-docker compose exec webauthn /var/www/html/bin/console  doctrine:migrations:migrate --env=prod
-```
+You should then get the apps initialised.
 
-Then you will need to provision the middleware config:
-```
-cd middleware
-sh middleware-push-config.sh
-sh middleware-push-whitelist.sh
-sh middleware-push-institution.sh
-```
-Then, bootstrap the SRAA. For this, you will need to have a Yubikey. Replace Yubikey_ID with the number that is printed on your yubikey. It should be 8 characters. If it is less, prepend it with 0's
+Initialise the middelware, gateway and webauthn database schema's and push the configuration of the middleware to the database:
+
+```shell
+./init-db.sh
 ```
-docker compose exec middleware  /var/www/html/bin/console middleware:bootstrap:identity-with-yubikey urn:collab:person:dev.openconext.local:admin dev.openconext.local "Your Name" Your@email nl_NL Yubikey_ID
+
+Then, bootstrap the SRAA user. For this, you will need to have a Yubikey. Replace `Yubikey_ID` in the command below with the serial number of your your Yubikey. This number is printed on your Yubikey. You can also get it by converting the first 12 characters from an OTP from your Yubikey from ModHex to decimal using <https://developers.yubico.com/OTP/Modhex_Converter.html>. The serial number must be at least 8 digits long. If it has less digits, prepend it with 0's.
+
+This associates the admin account with your Yubikey. The account has SRAA rights. You can then use your Yubikey to log in as user `admin` with password `admin`. Note that you must run this command before the first time you log in with the admin account to the SA or RA, otherwise this command will fail because the admin identity already exists.
+
+```shell
+docker compose exec middleware /var/www/html/bin/console middleware:bootstrap:identity-with-yubikey urn:collab:person:dev.openconext.local:admin dev.openconext.local "Your Name" Your@email en_GB Yubikey_ID
```

Comment:
Did you intentionally not move this to a script? I'm totally fine with having it in the README as it also ensures you need to remain engaged in order to not mis a step.